lib/salus/formatting.rb

```diff
@@ -4,22 +4,22 @@ module Formatting
     INDENT_STRING = (' ' * INDENT_SIZE).freeze
 
     TABLE_BORDER_CHARS = {
-      top_mid:      '┬',
-      top_left:     '┌',
-      top_right:    '┐',
-      bottom_mid:   '┴',
-      bottom_left:  '└',
+      top_mid: '┬',
```

Comment:
I'm not sure why rubocop flagged these. These should have been marked as an exception. I think for this file we prefer the way it is stylistically.

Reply:
Yeah, I ran rubocop -a

Reply:
hmm i typically do that but wonder why the ignores didn't trigger for you